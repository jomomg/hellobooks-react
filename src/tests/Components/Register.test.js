import React from 'react';
import Register from '../../components/register';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('<Register/>', ()=>{
    test('it renders successfully', ()=>{
        const register = shallow(<Register/>);
        expect(shallowToJson(register)).toMatchSnapshot()
    });
});