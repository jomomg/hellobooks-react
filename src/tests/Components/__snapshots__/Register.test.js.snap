// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Register/> it renders successfully 1`] = `
<Route>
  <Component />
</Route>
`;
